src/dmd/expressionsem.d

```diff
@@ -9462,19 +9462,28 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
         }
 
         Module mmath = loadStdMath();
-        if (!mmath)
+        version (none)
         {
-            //error("requires std.math for ^^ operators");
-            //fatal();
-            // Leave handling of PowExp to the backend, or throw
-            // an error gracefully if no backend support exists.
-            if (Expression ex = typeCombine(exp, sc))
+            // There's no gagging, and so loadStdMath will call fatal()
+            // if there's no std.math module available.
+            if (!mmath)
             {
-                result = ex;
+                //error("requires std.math for ^^ operators");
+                //fatal();
+                // Leave handling of PowExp to the backend, or throw
+                // an error gracefully if no backend support exists.
+                if (Expression ex = typeCombine(exp, sc))
+                {
+                    result = ex;
+                    return;
+                }
+                result = exp;
                 return;
             }
-            result = exp;
-            return;
+        }
+        else
+        {
+            assert(mmath !is null);
```

Comment:
`assert(0)` will do the job just fine, as:
```
if (cond) { ... } else { assert(!cond); }
```
seems faintly absurd.

Reply:
This is `version (none) { ... } else { assert(!cond); }`